### ISSUE TITLE
System properties RPC call returns empty value

### ISSUE THREAD

**Issue by Nutomic** [paritytech/subxt#888]
Im using the following code with subxt v0.27.1 to retrieve system properties:
```rust
let client: OnlineClient<PolkadotConfig> = ...;
let properties = client.rpc().system_properties().await;
dbg!(properties);
```

Strangely this returns an empty result `Ok({})`. In the web interface it gives the expected result:
```
{
  ss58Format: 42
  tokenDecimals: [
    18
  ]
  tokenSymbol: [
    Unit
  ]
}
```

Other RPC calls such as genesis_hash give the expected result which matches the result from the web interface:
```rust
let client: OnlineClient<PolkadotConfig> = ...;
let genesis = client.rpc().genesis_hash().await;
dbg!(genesis);
```

**Comment by jsdw** [comment]
The Subxt method expects back a result of the form `serde_json::Map<String, serde_json::Value>`.

I tried running this against a loval `polkadot` node:

```rust
let api = OnlineClient::<PolkadotConfig>::new().await?;
let props = api.rpc().system_properties().await?;
println!("{props:?}");
Ok(())
```

And I got back:

```
{"tokenDecimals": Number(10)}
```

Polkadot.js pointed at the same node making the same call returned back:

```
{"jsonrpc":"2.0","result":{"tokenDecimals":10},"id":4}
```

So what I see is consistent. I'd need more information to help reproduce; are you certain you pointed at the same node from Subxt and the web interface for instance?
